BlazorUI/Program.cs

```diff
@@ -0,0 +1,28 @@
+using System;
+using System.Collections.Generic;
```

Comment:
Codacy found an issue: [Remove this unnecessary 'using'.](https://app.codacy.com/manual/siriak/MoneyManager/pullRequest?prid=5088867)